experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+.columns {
+ display: flex;
+}
+
+.drag-drop {
+  display: inline-block;
+  position: absolute;
+  top: 0px;
+  width: 50px;
+  height: 50px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  transform: translate(0px, 0px);
+  z-index: 1;
+  background-size: 100%;
+}
+
+.logic-gate{
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.AND {
+  background-image: url("../images/AND.svg");
+}
+
+.NOT {
+  background-image: url("../images/NOT.svg");
+}
+
+.OR {
+  background-image: url("../images/OR.svg");
+}
+
+.XOR {
+  background-image: url("../images/XOR.svg");
+}
+
+.XNOR {
+  background-image: url("../images/XNOR.svg");
+}
+
+.NAND {
+  background-image: url("../images/NAND.svg");
+}
+
+.NOR {
+  background-image: url("../images/NOR.svg");
+}
+
+.FullAdder {
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.FullAdder:after {
+  content: "1-bit Full Adder";
+  margin: auto;
+  padding: 1.2rem 0rem;
+  font-weight: bold;
+  font-size: 1rem;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  text-align: center;
+}
+
+.HIGH {
+  text-align: center;
+  display: inline-block;
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  border: 0.2rem solid rgb(98, 178, 235);
+  transform: translate(0px, 0px);
+  z-index: 1;
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.LOW {
+  text-align: center;
+  display: inline-block;
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  border: 0.2rem solid rgb(236, 234, 90);
+  transform: translate(0px, 0px);
+  z-index: 1;
+  background-color: rgb(238, 235, 34);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.HIGH a,
+.LOW a {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  font-size: 1.2rem;
+  text-decoration: none;
+  color: black;
+  height: 100%;
+  width: 100%;
+}
+
+.HIGH p,
+.LOW p,
+.Output p {
+  position: absolute;
+  top: -30px;
+  align-items: center;
+  font-size: 1.3rem;
+}
+
+.HIGH a:hover,
+.LOW a:hover {
+  color: black;
+}
+
+.Output {

```

Comment:
Please do not use capital letters in identifiers unless they have some special meaning

Reply:
Yes, made them lowercase now.